spec/02-integration/09-hybrid_mode/09-config-compat_spec.lua

```diff
@@ -35,11 +35,11 @@ end
 local function get_plugin(node_id, node_version, name)
   local res, err = cluster_client({ id = node_id, version = node_version })
   assert.is_nil(err)
-  assert.is_table(res and res.config and res.config.plugins,
+  assert.is_table(res and res.config_table and res.config_table.plugins,
```

Comment:
Seems https://github.com/Kong/kong/pull/11523/files#diff-491970577eb085ed49987d03ef70a41bbd7643c4d0d8a0528d3ae761eac1cfa4R29 is never used.